error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -144,6 +144,28 @@ Mono<S> after(Mono<T> mono, S object) {
     }
   }
 
+  /**
+   * Prefer {@link Flux#take(long, boolean)} over {@link Flux#take(long)}.
+   *
+   * <p>In Reactor versions prior to 3.5.0, `Flux#take(long)` makes an unbounded request upstream,
+   * and is equivalent to `Flux#take(long, false)`. In 3.5.0, the behavior of `Flux#take(long)` will
+   * change to that of `Flux#take(long, true)`.
+   *
+   * <p>The intent with this Refaster rule is to get the new behavior before upgrading to Reactor
+   * 3.5.0.
+   */
+  static final class FluxTake<T> {
```

Comment:
This is one of the few (perhaps the first?) Refaster rules that is intentionally not behavior preserving. A good opportunity to use the `@Description` and `@Severity` annotations introduced in #255. :smile: